src/libraries/System.Private.CoreLib/src/System/Collections/Generic/ICollectionDebugView.cs

```diff
@@ -11,14 +11,7 @@ internal sealed class ICollectionDebugView<T>
 
         public ICollectionDebugView(ICollection<T> collection)
         {
```

Comment:
```suggestion
        {
#if NET
```

https://github.com/dotnet/runtime/blob/8886107370b1ff15662dd38f3b3a5546188842e7/src/libraries/System.Text.Json/src/System.Text.Json.csproj#L362

Reply:
apologize didn't see that.